bin/cml-runner.js

```diff
@@ -326,9 +352,15 @@ const opts = yargs
     'idle-timeout',
     'Time in seconds for the runner to be waiting for jobs before shutting down. Setting it to 0 disables automatic shutdown'
   )
-  .default('name', RUNNER_NAME)
-  .describe('name', 'Name displayed in the repository once registered')
-
+  .default('name')
+  .describe('name', 'Name displayed in the repository once registered cml-{ID}')
+  .coerce('name', (val) => val || RUNNER_NAME)
+  .boolean('retry')
+  .default('retry', RUNNER_RETRY)
+  .describe(
+    'retry',
+    'Automatically retry jobs terminated due to runner disposal or timeout (72 hours on Github)'
```

Comment:
```suggestion
    'Whether to restart workflows terminated due to instance disposal or GitHub Actions timeout'
```

Reply:
also, I don't know if we should be inverting this option (i.e. retry by default and provide `--no-retry` to disable)?

Reply:
> also, I don't know if we should be inverting this option (i.e. retry by default and provide --no-retry to disable)?

I love it. I was trying to be conservative but 💯

Reply:
kewl looks like we all are happier with `--no-retry`.